src/web/templates/base.html

```diff
@@ -43,7 +43,7 @@
     <!-- my styles -->
     <link href="../css/styles.css" rel="stylesheet">
 
-    <title>CRIPT Excel Uploader - version 1.0.2</title>
+    <title>CRIPT Excel Uploader - version 1.0.1</title>
```

Comment:
10 commits; one line changed, I suggest highly to do rebase instead of merge.

Reply:
Can you tell me a bit more? why rebase?